docs/adding_a_benchmark.md

```diff
@@ -0,0 +1,40 @@
+## Adding a new Benchmark 
+
+MTEB covers a wide variety benchmark which are all presented in the public [leaderboard](https://huggingface.co/spaces/mteb/leaderboard). However many languages or domains are still missing benchmark and we welcome contributions.
+
+To add a new benchmark you will need to:
+
+1) [Implement the tasks](adding_a_dataset.md) that you want to evaluate on, or find then in the existing list of tasks.
+2) Implement the benchmark in the [`benchmark.py`](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/benchmarks/benchmarks.py) file and submit it as a PR
+
+This is easy to do 
+```python
+tasks = mteb.get_tasks(tasks=[] ...) # fetch the tasks you want to include in your benchmark
+
+MY_BENCHMARK = Benchmark(
+    name="Name of your benchmark",
+    tasks=tasks,
+    description="This benchmark tests y, which is important because of X",
+    reference="https://relevant_link_eg_to_paper.com",
+    citation="A bibtex citation if relevant",
+)
+```
+
+3) Run a representative set of models on benchmark and submit the results you do this by
+<!-- TODO: we should probably create seperate page for how to submit results -->
+1. Open a PR in https://hf.co/datasets/mteb/results with:
+- All results added in existing model folders or new folders
+- Updated paths.json (see snippet results.py)
+<!-- TODO: ^check if this is still required, if so we should probably update it, if not we should remove it once the new leaderboard is lie -->
+- If adding any new models, their names added to results.py
+- If you have access to all models you are adding, you can also [add results via the metadata](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_model.md) for all of them / some of them
+1. Open a PR at https://huggingface.co/spaces/mteb/leaderboard modifying app.py to add your tab:
+- Add any new models & their specs to the global lists
+- Add your tab, credits etc to where the other tabs are defined
+- If you're adding new results to existing models, remove those models from `EXTERNAL_MODEL_RESULTS.json` such that they can be reloaded with the new results and are not cached.
+- You may also have to uncomment `, download_mode='force_redownload', verification_mode="no_checks")` where the datasets are loaded to experiment locally without caching of results
+- Test that it runs & works locally as you desire with python app.py, **please add screenshots to the PR**
```

Comment:
Is it only for adding a new tab to the leaderboard, or every time a new model is added?
If it's the latter, then 3 PRs are needed to add a new model: MTEB repo, results repo, and leaderboard repo.